etrago/tools/utilities.py

```diff
@@ -203,18 +209,19 @@ def geolocation_buses(self):
         c_bus1 = network.buses.loc[network.links.loc[link, "bus1"], "country"]
         network.links.loc[link, "country"] = "{}{}".format(c_bus0, c_bus1)
 
-    return network
+    return
 
 
-def buses_by_country(self):
+def buses_by_country(network, con):
```

Comment:
I am still wondering if it is the best option to remove functions from the object. I will think about it and reply again.